website/blog/2022-04-19-complex-deduplication.md

```diff
@@ -88,7 +88,9 @@ To build our `grain_id` key, we use the pure gold of the *[dbt_utils package](ht
 ```sql
 {% macro build_key_from_columns(table_name, exclude=[]) %}
 
-{% set cols = {{ dbt_utils.star(from=ref('table_name'), except = exclude) }} %}
+{% set cols =  dbt_utils.star(from=table_name, except = exclude)  %}
```

Comment:
This may be my misunderstanding, but doesn't the `from` argument need to be passed in a ref/source? Is this a way to generalize it out?